### PR TITLE
Glasses containers #10

### LINKED ISSUE #10: GlassesContainer and SingleGlassesContainer

### PR DESCRIPTION
**Pull request by tashadesai**
closes #10

### CODE REVIEW

File: app/reducers/glasses.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+})
+
+
+export const getGlasses = () =>
+  dispatch =>
+    axios.get('/api/glasses')
+      .then((glasses) => dispatch(fetchAllGlasses(glasses.data)))
+      .catch(console.error)
+
+export const getSingleGlasses = (glassesId) =>
```
Comment (pimpila) [inline app/reducers/glasses.jsx:45]: in keeping w/ logic we have for seasons, should this just filter the store.glasses.allGlasses?

File: app/components/SingleGlasses.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <h3>{props.selectedGlasses.material}</h3>
+      <h3>{props.selectedGlasses.price}</h3>
+      <h3>{props.selectedGlasses.description}</h3>
+      <h3>{props.selectedGlasses.quantity}</h3>
+    </div>
+  )
+}
+
+
+export default connect(
```
Comment (pimpila) [inline app/components/SingleGlasses.jsx]: Call this container since it's connected?

File: app/containers/GlassesContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+//USELESS
```
Comment (pimpila) [inline app/containers/GlassesContainer.jsx]: delete this file?

**Review by MarcosRS — COMMENTED** [review]
Reviewed with Pim

**Pull request merged**